test/jdk/sun/util/logging/PlatformLoggerTest.java

```diff
@@ -132,6 +133,7 @@ private static void checkLogger(String name, Level level) {
             throw new RuntimeException("Invalid level for logger " +
                 logger.getName() + " " + logger.getLevel());
         }
+        Reference.reachabilityFence(logger);
```

Comment:
Hello Daniel, I don't follow why the reachabilityFence would be needed here for the `logger`. Wouldn't the `logger` be already "alive" at this line in the method?

Reply:
Yes - it's not strictly needed here - I added it for symmetry in case we add more checks to the method.
I also suspect that the only place where it's needed is at line 179, since all other loggers are saved into statics. 
But better safe than sorry :-)

Reply:
That sounds fine to me then.